install/kubernetes/helm/istio/charts/tracing/values.yaml

```diff
@@ -31,9 +31,14 @@ podAntiAffinityTermLabelSelector: []
 jaeger:
   hub: docker.io/jaegertracing
   image: all-in-one
-  tag: 1.9
+  tag: 1.12
```

Comment:
Sorry just had a chance to look at this - the latest version is currently 1.13.1.

Reply:
@objectiser I knew the latest version before. I think the badger feature should be mature in 1.12. I am not sure we should go with the 1.13.1 because it is just out recently.